client/src/components/NavBar/AuthBars/LoggedInBar.tsx

```diff
@@ -5,28 +5,63 @@ import { Link } from 'react-router-dom';
 import useStyles from './useStyles';
 import AuthMenu from '../../AuthMenu/AuthMenu';
 import NotificationLink from '../../NotificationDisplay/NotificationDisplay';
+import TextTransition, { presets } from 'react-text-transition';
+import { useEffect, useState } from 'react';
 
 const LoggedInBar = (): JSX.Element => {
   const classes = useStyles();
+  const [showText1, setShowText1] = useState<boolean>(true);
+  const [showText2, setShowText2] = useState<boolean>(true);
+
+  useEffect(() => {
+    setInterval(() => {
+      setShowText1(false);
+      setTimeout(() => {
+        setShowText2(false);
+      }, 3000);
+    }, 6000);
+  }, [setShowText1, setShowText2]);
+
+  useEffect(() => {
+    if (!showText1) {
+      setShowText1(true);
+    }
+  }, [showText1]);
+
+  useEffect(() => {
+    if (!showText2) {
+      setShowText2(true);
+    }
+  }, [showText2]);
 
   return (
     <Grid container className={classes.navButtons}>
       <Grid item>
-        <NotificationLink />
+        <NotificationLink>
+          <Typography variant="h3">
+            <TextTransition text={showText2 ? 'Notifications' : ''} springConfig={presets.default} />
```

Comment:
Optional: These transition effects look cool but I would suggest reserving them only for the logo (also, the may need to be disabled on mobile views) 😄

Reply:
OK, I disable it on mobiles